webapp/src/modules/analytics/sentry.ts

```diff
@@ -0,0 +1,20 @@
+import { init, BrowserTracing, Replay } from '@sentry/react'
+import { config } from '../../config'
+
+init({
+  environment: config.get('ENVIRONMENT'),
+  release: '6.6.7', // TODO: Replace with the app's version
```

Comment:
You might be able to use the environment variable set at the `webapp/scripts/prebuild.js` script to get the release version as `process.env.REACT_APP_WEBSITE_VERSION`.
```suggestion
  release: process.env.REACT_APP_WEBSITE_VERSION
```